### ISSUE TITLE
photos plugin: can we explicitly set the output folder?

### ISSUE THREAD

**Issue by beleriandcrises** [getpelican/pelican-plugins#994]
Hello everyone.

In a website I'm building I have two languages, `en` and `it`. Now, I love the work that photos does as a plugin, and it works properly with the i18n, but seems to me a huge waste of resources to have both `/photos` and `/it/photos` with the same content, so I'd like to set the same photos folder for both.

Just like I've done for the images folder, I'd like to be able to do something like:

```
PHOTOS_GALLERIES='photos'

I18N_SUBSITES = {
    'it': {
        'PHOTOS_GALLERIES= '../photos',
        }
    }
```

Would it be doable?
I'm studying the plugin right now to see if I can do it myself, but I'm not a good programmer =D

Thanks!

**Comment by justinmayer** [comment]
Any thoughts, @smartass101?

**Comment by smartass101** [comment]
There was some related discussion in getpelican/pelican#1042 which aimed at using symlinks instead of copying. It appears that it has been implemented since then in getpelican/pelican#1982. So I'd look in the last issue on how to use symbolic or hard links for static files. If you'd like to use that functionality, you would have to use the pelican master branch (this functionality hasn't been included in a release yet).

**Comment by justinmayer** [comment]
Thanks for the insight, Ondrej. That could indeed be a way to handle this.

@beleriandcrises: If you need further assistance with this, please reach out via the Pelican IRC channel, as noted in the [How to Get Help](http://docs.getpelican.com/en/stable/contribute.html#how-to-get-help) section of the documentation. Thanks!
